docs/setup/upgrade/upgrade-migrations.asciidoc

```diff
@@ -65,39 +69,18 @@ For {kib} < 7.5.1, if the task manager index is set to `.tasks` with the configu
 [[resolve-migrations-failures]]
 ==== Resolving migration failures
 
-If {kib} terminates unexpectedly while migrating a saved object index, manual intervention is required before {kib} will attempt to perform the migration again. Follow the advice in (preventing migration failures)[preventing-migration-failures] before retrying a migration upgrade.
-
-As mentioned above, {kib} will create a migration lock for each index that requires a migration by creating a new `.kibana_N+1` index. For example: if the `.kibana_task_manager` alias is pointing to `.kibana_task_manager_5` then the first {kib} that succeeds in creating `.kibana_task_manager_6` will obtain the lock to start migrations.
-
-However, if the instance that obtained the lock fails to migrate the index, all other {kib} instances will be blocked from performing this migration. This includes the instance that originally obtained the lock, it will be blocked from retrying the migration even when restarted.
-
-[float]
-===== Retry a migration by restoring a backup snapshot:
-
-1. Before proceeding ensure that you have a recent and successful backup snapshot of all `.kibana*` indices.
-2. Shutdown all {kib} instances to be 100% sure that there are no instances currently performing a migration.
-3. Delete all saved object indices with `DELETE /.kibana*`
-4. Restore the `.kibana* indices and their aliases from the backup snapshot. See {es} {ref}/modules-snapshots.html[snapshots]
-5. Start up all {kib} instances to retry the upgrade migration.
-
-[float]
-===== (Not recommended) Retry a migration without a backup snapshot:
-
-1. Shutdown all {kib} instances to be 100% sure that there are no instances currently performing a migration.
-2. Identify any migration locks by comparing the output of `GET /_cat/aliases` and `GET /_cat/indices`. If e.g. `.kibana` is pointing to `.kibana_4` and there is a `.kibana_5` index, the `.kibana_5` index will act like a migration lock blocking further attempts. Be sure to check both the `.kibana` and `.kibana_task_manager` aliases and their indices.
-3. Remove any migration locks e.g. `DELETE /.kibana_5`. 
-4. Start up all {kib} instances.
+If {kib} terminates unexpectedly while migrating a saved object index it will automatically attempt to perform the migration again once the process has restarted. If upgrade migrations fail repeatedly, follow the advice in (preventing migration failures)[preventing-migration-failures]. Unlike previous versions of {kib}, v7.12+ does not require deleting any indices to release a failed migration lock. Once the root cause for the migration failure has been addressed, {kib} will automatically retry the migration without any further intervention.
```

Comment:
Maybe we should add a very clearly worded warning that in 7.12+ it is not recommended to delete any indices or aliases without first contacting Elastic Support.

We should also add a note that if you're dealing with a version prior to 7.12 to refer to the older docs. Not sure if our doc system supports linking to previous versions, but if so that'd be nice.

Reply:
I used stronger wording in 927b60a3185